yt/fields/vector_operations.py

```diff
@@ -104,18 +104,34 @@ def _relative_vector(field, data):
         )
 
 
-def create_los_field(registry, basename, field_units, ftype="gas", slice_info=None):
+def create_los_field(
+    registry,
+    basename,
+    field_units,
+    ftype="gas",
+    slice_info=None,
+    *,
+    sampling_type="local",
+):
     axis_order = registry.ds.coordinates.axis_order
 
+    # Here we need to check if we are a particle field, so that we can
+    # correctly identify the "bulk" field parameter corresponding to
+    # this vector field.
+    if sampling_type == "particle" and basename.startswith("particle_"):
```

Comment:
This condition is now redundant with `str.removeprefix`
```suggestion
    if sampling_type == "particle":
```